cellseg_gsontools/helper.py

```diff
@@ -0,0 +1,50 @@
+import cv2
```

Comment:
helper files could be moved under `utils` folder/module for better repo structure